lib_methods.rb

```diff
@@ -0,0 +1,118 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+# rubocop:disable Metrics/ModuleLength
+module Actions
+  def list_books
+    if @books.empty?
+      puts 'No books available'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'No person available'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)?: '
+    user_input = gets.chomp.to_i
+    case user_input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else puts 'Please select 1 or 2'
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Does student have parent permission? [Y/N]: '
+    permission = gets.chomp
+    new_student = Student.new(age, permission, name)
```

Comment:
- [ ] I think this line of code should've been placed in the 'create_person()' method since a student is a person